modules/oe_editorial_corporate_workflow/oe_editorial_corporate_workflow.module

```diff
@@ -139,13 +140,35 @@ function oe_editorial_corporate_workflow_form_content_moderation_entity_moderati
 function _oe_editorial_corporate_workflow_moderation_submit(array &$form, FormStateInterface &$form_state): void {
   /** @var \Drupal\oe_editorial_corporate_workflow\Services\ShortcutRevisionHandler $shortcut_handler */
   $shortcut_handler = \Drupal::service('oe_editorial_corporate_workflow.shortcut_transition_handler');
-  $latest_revision = $shortcut_handler->createShortcutRevisions($form_state->getValue('new_state'), $form_state->get('entity'), $form_state->getValue('revision_log'));
```

Comment:
I don't see this API used anymore so..should we keep it? Including the recursive `createShortcutRevisions` protected method. We can deprecate it and remove it in the next version or keep it in case it might be useful for others? But with the batch in place, why would anyone use that?

Reply:
I kept it for backwards compatibility, I don't know if users have overridden the class and adding their own logic on it.
We could mark it as deprecated though.

Reply:
Let's mark it deprecated because it is in fact replaced by the batch way.